tests/testthat/adtteSpec/tests/testthat/test-shinytest2.R

```diff
@@ -0,0 +1,51 @@
+library(shinytest2)
```

Comment:
hm I guess you tried already, anyway asking - is there a way to have this file within the `tests/testthat` directory (so 3 folders up) or within the `tests/testthat/adtteSpec` directory? it feels like there are a lot of folders wrapped in each other now...